pydis_site/apps/api/serializers.py

```diff
@@ -1,4 +1,5 @@
 """Converters from Django models to data interchange formats and back."""
+import logging
```

Comment:
I've accidentally left this logging import here; I'll remove it when pushing the commit that straightens out the migration conflict that #269 will cause.